src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task4.java

```diff
@@ -0,0 +1,17 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+import java.util.Scanner;
+
+public class Task4 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a last number please:");
+        int sum = 0; //our first sum, initializing our sum
+        int LastNumber = scanner.nextInt();
```

Comment:
Please change  int LastNumber to  int lastNumber

Reply:
Done, thank you